authors/apps/articles/models.py

```diff
@@ -129,3 +131,67 @@ class Favorite(models.Model):
         related_name='favorites')
     article_id = models.ForeignKey(
         Article, on_delete=models.CASCADE, related_name='favorites')
+
+
+class Tag(models.Model):
+    tag = models.CharField(max_length=100, db_index=True, unique=True)
+    slug = models.SlugField(max_length=240, db_index=True, unique=True)
+
+    def save(self, *args, **kwargs):
+        if not self.slug:
+            self.slug = slugify(self.tag)
+        super().save(*args, **kwargs)
+
+    def __str__(self):
+        return self.slug
+
+    objects = models.Manager()
+
+    def _create_tag(self, tag_name):
```

Comment:
@Etomovich, why does this method name start with `_`?

Reply:
@abulojoshua1, it implies it is an internal method